gensim/scripts/make_wiki.py

```diff
@@ -1 +0,0 @@
-make_wikicorpus.py
```

Comment:
See my update to this PR: https://github.com/piskvorky/gensim/pull/615

@rutum Something weird happened when you merged the dev branch into your branch, and you ended up replacing all the symlinks with their actual files.